_config.yml

```diff
@@ -56,15 +56,16 @@ defaults:
       type_of_guidance: "Test Rule"
       rule_meta:
         rule_type: "atomic rule"
-      sidebar:
-        footer:
-          content: Test Rules describe ways to test conformance to WCAG success criteria. They are <a href="../about">for evaluation</a>.
-      standalone_resource_nav_links:
-        - name: All Rules
-          ref: /standards-guidelines/act/rules/
-        - name: About Rules
+      sidebar: true
+      standalone_resource_header:
+        title: WCAG 2 Test Rules
+        subtitle: For developers of evaluation tools and test methodologies
+        link:
+          title: About WCAG 2 Test Rules
           ref: /standards-guidelines/act/rules/about/
-      standalone_resource_doc_name: "WCAG 2 Test Rules"
+      standalone_resource_pager:
```

Comment:
Is this the navbar content? Two things about that:

1. Can we add back the link to the "About Rules" page?
2. The styling of the navbar doesn't look right. The active element doesn't have the white background, and when it's not active, this just looks like an inline link, rather than a tab.

Reply:
The nav bar yaml was refactored. The link to about page is on line 65. We dropped the tabs an iteration back if that's what you mean? 

You can also add paging like the supplemental guidance if you wish...
![image](https://user-images.githubusercontent.com/618922/159477666-4f353799-7b27-4ec0-980e-48732c5cc9af.png)

Reply:
> Can we add back the link to the "About Rules" page?

It's in the header to be consistent. Additional input welcome in new specific issue https://github.com/w3c/wcag-act-rules/issues/94